app/css/components/insiders-prompts.css

```diff
@@ -1,18 +1,24 @@
+.c-react-wrapper-insiders-status {
+    @apply min-h-[160px];
+}
```

Comment:
This is the correct way to stop the flash. It needs to go on the `c-...` level component, so the styling applies before react hydrates it.